README.md

```diff
@@ -30,6 +30,8 @@ On some Linux distributions there are also distribution packages available:
  * Debian Sid (unstable): http://packages.debian.org/sid/colobot
  * Arch Linux (AUR): https://aur.archlinux.org/packages/colobot-gold
  * openSUSE: http://software.opensuse.org/download.html?project=games&package=colobot
+ * Fedora: https://src.fedoraproject.org/rpms/colobot
+   - `sudo dnf install colobot`
```

Comment:
You should stick to link only rather than full install command - other links do not have them.

Reply:
I could, however, on that page there is no instructions how to install the package. It would be much more intuitive for the users to leave it as:
```
 * Fedora: sudo dnf install colobot
```

(without the link at all).

WDYT?

Reply:
I agree with MrSimbax #2: https://github.com/colobot/colobot/pull/1292#pullrequestreview-406767494